analysis/scripts/2233-power-calculations.Rmd

```diff
@@ -214,40 +226,49 @@ simulation_01 <- function(
   treatment = complete_ra(N = in_population_size,prob = in_prop_assigned_treatment),
   outcome_0 = outcome_0,
   outcome_1 = case_when(
-    effect*treatment + outcome_0 < 0 ~ 0,
-    effect*treatment + outcome_0 >= 0 ~ effect*treatment + outcome_0
+    effect + outcome_0 < 0 ~ 0,
+    effect + outcome_0 >= 0 ~ effect + outcome_0
   ),
-  outcome_obs = treatment*outcome_1 + (1-treatment)*outcome_0
+  true_effect = outcome_1 - outcome_0,
```

Comment:
I wanted to record the true effect because I wanted to be sure that the tests for which we are calculating power (aka 1 - false negative rate) also have a correct false positive rate.